ddtrace/tracer/option.go

```diff
@@ -482,6 +554,56 @@ func (c *config) loadAgentFeatures() {
 	}
 }
 
+// MarkIntegrationImported labels the given integration as imported
+func MarkIntegrationImported(integration string) bool {
+	s, ok := contribIntegrations[integration]
+	if !ok {
+		return false
+	}
+	s.imported = true
+	contribIntegrations[integration] = s
+	return true
+}
+
+func (c *config) loadContribIntegrations() {
+	integrations := map[string]integrationConfig{}
+	deps, ok := debug.ReadBuildInfo()
+	for _, s := range contribIntegrations {
+		integrations[s.name] = integrationConfig{
+			Instrumented: s.imported,
+		}
+	}
+	if !ok {
+		c.integrations = integrations
+		return
+	}
+	for _, d := range deps.Deps {
+		p := d.Path
+		// special use case, since gRPC does not update version number
+		if p == "google.golang.org/grpc" {
+			re := regexp.MustCompile(`v(\d.\d)\d*`)
```

Comment:
This logic is a bit complicated / has an interesting regex, is this covered by any tests explicitly? (Also what's the coverage report say about this function / these new files?